test/service_instance_and_binding_test/service_binding_test/service_binding_test.go

```diff
@@ -239,6 +239,90 @@ var _ = DescribeTestsFor(TestCase{
 				ctx.CleanupAdditionalResources()
 			})
 
+			Describe("Get Parameters, service does not exist", func() {
+				It("should return an error", func(){
+						ctx.SMWithOAuthForTenant.GET(web.ServiceBindingsURL + "/" + bindingID+"/params").Expect().
+							Status(http.StatusNotFound).JSON().Object().Value("error").String().Equal("NotFound")
+
+				})
+
+			})
+			Describe("Get parameters, service binding does not exist", func() {
```

Comment:
should be when

Reply:
i deleted this code, i forgot to delete. i resturcted the tests